doc/code_snippets/snippets/connectors/go/hello.go

```diff
@@ -0,0 +1,132 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"github.com/tarantool/go-tarantool/v2"
+	_ "github.com/tarantool/go-tarantool/v2/datetime"
+	_ "github.com/tarantool/go-tarantool/v2/decimal"
+	_ "github.com/tarantool/go-tarantool/v2/uuid"
+	"time"
+)
+
+func main() {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+	defer cancel()
+	dialer := tarantool.NetDialer{
+		Address:  "127.0.0.1:3301",
+		User:     "sampleuser",
+		Password: "123456",
+	}
+	opts := tarantool.Opts{
+		Timeout: time.Second,
+	}
+
+	conn, err := tarantool.Connect(ctx, dialer, opts)
+	if err != nil {
+		fmt.Println("Connection refused:", err)
+		return
+	}
+
+	// Interacting with the database
+	// Insert data
+	tuples := [][]interface{}{
+		{1, "Roxette", 1986},
+		{2, "Scorpions", 1965},
+		{3, "Ace of Base", 1987},
+		{4, "The Beatles", 1960},
+	}
+	var futures []*tarantool.Future
+	for _, tuple := range tuples {
+		request := tarantool.NewInsertRequest("bands").Tuple(tuple)
+		futures = append(futures, conn.Do(request))
+	}
+	fmt.Println("Inserted tuples:")
+	for _, future := range futures {
+		result, err := future.Get()
+		if err != nil {
+			fmt.Println("Got an error:", err)
+		} else {
+			fmt.Println(result)
+		}
+	}
+
+	// Select by primary key
+	data, err := conn.Do(
+		tarantool.NewSelectRequest("bands").
+			Limit(10).
+			Iterator(tarantool.IterEq).
+			Key([]interface{}{uint(1)}),
+	).Get()
+	if err != nil {
+		fmt.Println("Got an error:", err)
+	}
+	fmt.Println("Tuple selected the primary key value:", data)
```

Comment:
**by** the key value?
Same for secondary key (line 77)

Reply:
omg, thanks!